gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFSInputStream.java

```diff
@@ -126,6 +134,7 @@ public synchronized int read(@Nonnull byte[] buf, int offset, int length) throws
 
   @Override
   public synchronized void seek(long pos) throws IOException {
+    checkNotClosed();
```

Comment:
should we add the same check for getPos() as well?

Reply:
Done.